templates/parachain/node/Cargo.toml

```diff
@@ -22,82 +19,31 @@ jsonrpsee = { features = ["server"], workspace = true }
 futures = { workspace = true }
 serde_json = { workspace = true, default-features = true }
 docify = { workspace = true }
+color-print = { workspace = true }
+
+polkadot-sdk = { workspace = true, features = ["experimental", "node"] }
```

Comment:
Why are we enabling a experimental feature in our templates? There is something fishy again..

Reply:
I would soon remove this flag from the frame umbrella crate, rest assured.